pooch/tests/test_downloaders.py

```diff
@@ -522,6 +528,7 @@ def test_download_url(self, httpserver, api_version, api_response):
         "api_response",
         [legacy_api_response, new_api_response],
     )
+    @pytest.mark.network
```

Comment:
Same case with this test, which also makes use of `httpserver`. Do you face any problem if you run this test without connection?
```suggestion
```

Reply:
Dito.